Exiled.API/Features/Player.cs

```diff
@@ -1184,6 +1186,31 @@ public void ShowHint(string message, float duration = 3f)
             HintDisplay.Show(new TextHint(message, parameters, null, duration));
         }
 
+        /// <summary>
+        ///  Disables all status affects on the player.
+        /// </summary>
+        public void DisableAllEffects()
+        {
+            foreach (KeyValuePair<Type, PlayerEffect> effect in ReferenceHub.playerEffectsController.AllEffects)
+            {
+                effect.Value.ServerDisable();
+            }
+        }
+
+        /// <summary>
+        /// Enables a status effect on this player.
+        /// </summary>
+        /// <param name="effect">The <see cref="PlayerEffect"/> to enable.</param>
+        /// <param name="duration">The amount of time the effect will be active for.</param>
+        public void EnableEffect(PlayerEffect effect, float duration = 0f) => ReferenceHub.playerEffectsController.EnableEffect(effect, duration);
```

Comment:
The best implementation is `EnableEffect<T>(float duration = 0f, bool 
addDurationIfActive = false) where T : PlayerEffect` because literally nobody will cache an effect object

Reply:
true